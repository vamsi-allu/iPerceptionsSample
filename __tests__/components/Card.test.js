import 'jsdom-global/register';
import React from 'react';
import Card from '../../components/Card';
import Enzyme, {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

describe('Card Component', () => {
  it('renders correctly enzyme', () => {
    const props = {style: {}};
    const wrapper = shallow(<Card {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('check the view is loaded', () => {
    const props = {style: {}};
    const component = shallow(<Card {...props} />);

    expect(component.find('View').getElements()).toBeDefined();
  });
});
