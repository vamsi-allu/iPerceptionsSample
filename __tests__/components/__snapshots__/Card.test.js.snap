// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders correctly enzyme 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 10,
      "padding": 20,
    }
  }
/>
`;
